resources/js/components/AvailableCourses.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div id="alignment">
+    <div class="AvalaibleCourses">
+      <div class="row">
+        <div v-for="x in 8" :key="x" class="col-md-3">
+          <div class="card" style="width: 18rem">
+            <img src="images/course_image.png" class="card-img-top" alt="" />
+            <div class="card-body">
+              <a href="#" class="btn btn-secondary">Applied Science</a>
+              <h5 class="card-title">Physical Science</h5>
+              <p class="card-text">Dr Kawu</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "AvalaibleCourses",
+};
+</script>
+
+<style>
```

Comment:
Can we please scope this styles that way what is written here doesn't affect other components. If its a global style, then it should go to our `app.scss` file.